cmd/osm-controller/osm-controller.go

```diff
@@ -300,6 +300,9 @@ func startMetricsStore() {
 		metricsstore.DefaultMetricsStore.K8sMonitoredNamespaceCount,
 		metricsstore.DefaultMetricsStore.K8sMeshPodCount,
 		metricsstore.DefaultMetricsStore.K8sMeshServiceCount,
+		metricsstore.DefaultMetricsStore.K8sMeshServiceAccountCount,
+		metricsstore.DefaultMetricsStore.K8sMeshIngressCount,
+		metricsstore.DefaultMetricsStore.K8sMeshEndpointCount,
```

Comment:
All of this can be queried using the `K8sAPIEventCounter` metric which is labeled with the event type. We shouldn't be adding a counter for each type of event.